src/strings.json

```diff
@@ -103,7 +103,7 @@
 	"WebClipper.Preview.Header.AttachPdfFile": "Attach PDF File",
 	"WebClipper.Preview.Header.PdfTooLargeToAttach": "PDF too large to attach",
 	"WebClipper.Preview.Header.PdfDistributePagesCheckboxLabel": "Create page for each PDF page",
-	"WebClipper.Preview.Header.PdfInvalidPageRange": "Invalid page range.",
+	"WebClipper.Preview.Header.PdfInvalidPageRange": "We couldn\u0027t find page",
```

Comment:
I think @leakymemory suggested a while back doing something like "We couldn\u0027t find page {0}" where {0} is replaced with the page number. I'm not sure if all localized strings will keep the page number at the end of the string, so this is our safest bet.